opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -69,6 +69,8 @@ class LogDroppedAttributesWarning(UserWarning):
 
 warnings.simplefilter("once", LogDroppedAttributesWarning)
 
+Any = Union[str, bool, int, float, bytes, List["Any"], Dict[str, "Any"], None]
```

Comment:
To avoid confusion with the python Any I would name it something else so it's explicit what it is, `OtelAny` maybe? Then I think this type should be in some common package.

Reply:
I did this .
what other things would you suggest to do

Reply:
Reading the thread, In java they created as `AnyValue`

Reply:
@emdneto 
changed it to _`AnyValue`_ 
would you review it